### PR TITLE
Stellar settings: be aware if we can send tx, do not show "Set Inflation" button if not

### PR DESCRIPTION
**Pull request by zapu**
Present the distinction between "funded accounts" and not to the frontend, as well as another field that says whether there is enough to make a tx or not. Use this info to hide "Set inflation" button if this account can't set any inflation yet.

### CODE REVIEW

**Comment by zapu** [comment]
@patrickxb can you review core side? thanks!

File: protocol/avdl/stellar1/local.avdl [hunk 1]
```diff
@@ -14,6 +14,8 @@ protocol local {
     string seqno;               // the stellar sequence number for this account
     CurrencyLocal currencyLocal;// user configurable, defaults to USD and related details
     AccountMode accountMode;    // mobile only vs all devices
+    boolean isFunded;           // does the account have min amount of XLM (exists on the network)
+    boolean canMakeTx;          // does the account have enough XLM for a transaction fee
```
Comment (zapu) [inline protocol/avdl/stellar1/local.avdl]: I'm fine with renaming this field, I couldn't come up with a good name

Comment (patrickxb) [reply protocol/avdl/stellar1/local.avdl]: I don't have much better....`canSubmitTx`, `isFundedEnoughForTx`, `hasAvailableBalance`

File: shared/wallets/wallet/settings/index.js [hunk 2]
```diff
@@ -177,12 +178,19 @@ class AccountSettings extends React.Component<SettingsProps> {
...
+                <Kb.Button
+                  type="Secondary"
+                  label={props.inflationDestination ? 'Change' : 'Set up'}
+                  onClick={props.onSetupInflation}
+                  style={styles.setupInflation}
+                />
+              )}
+              {!props.canSubmitTx && (
+                <Kb.Text type="BodySmall">
+                  Your account needs a minimum balance to set an inflation destination.
```
Comment (buoyad) [inline shared/wallets/wallet/settings/index.js]: 'a minimum balance' -> 'more funds'

**Pull request merged**